src/tokens/selectors.ts

```diff
@@ -467,6 +467,15 @@ export const sortedTokensWithBalanceOrShowZeroBalanceSelector = createSelector(
     })
 )
 
+export const sortedCkesandCusdSelector = createSelector(
+  (state: RootState, networkIds: NetworkId[]) => tokensListSelector(state, networkIds),
```

Comment:
get all of the tokens (since Jacob's changes make it so only cKES and cUSD can be returned